pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = ['setuptools >= 61.0']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'occameracontrol'
```

I know, I named the package :grinning: but…

Package names in Python are pretty restricted plus I didn't want it to be too long. That's why I settled for `occameracontrol`. The name set here will appear on pypi.org and in `pip`. It's also less restricted. Therefor, I suggest we use a better name. My suggestion would still be `opencast-camera-control`. What do you think?

That would e.g. result in an installation via:
```
❯ pip install opencast-camera-control
```

```suggestion
name = 'opencast-camera-control'
```